lib/jekyll-contentful-data-import/data_exporter.rb

```diff
@@ -17,9 +17,24 @@ def initialize(name, entries, config = {})
 
       def run
         setup_directory
+        puts config
+        export_fn = config['multiple_files'] ? export_by_content_type : export
+        export_fn
```

Comment:
`export_fn =` is not needed, as well as the following line, as it would already return the value of either `#export_by_content_type` or `#export`